.github/workflows/ci.yml

```diff
@@ -42,6 +42,21 @@ jobs:
           project-token: ${{ secrets.CODACY_PROJECT_TOKEN }}
           coverage-reports: ./target/scala-3.3.0/coverage-report/cobertura.xml
 
+  test-docs-build:
+    name: Test docs
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v3
+      - name: Install requirements (pip, npm, apt)
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r docs/requirements.txt
+          npm install --global typedoc
```

Comment:
is typedoc needed for the OpenAPI stuff? or what does it do for us here?

Reply:
see https://github.com/dasch-swiss/dsp-ingest/pull/26#discussion_r1250709780